client/coral-embed-stream/src/containers/Stream.js

```diff
@@ -112,8 +112,8 @@ class StreamContainer extends React.Component {
 
   componentDidMount() {
     if (this.props.previousTab) {
-      this.props.data.refetch();
-      this.props.setCommentCountCache(0);
+      this.props.data.refetch()
+        .then(() => this.props.setCommentCountCache(0));
```

Comment:
Why do you set it to `0`? Does this work? I think it should be set to `asset.commentCount` from the new query result. Do you have access to it inside this callback?